PostgresV11.md

```diff
@@ -5,28 +5,8 @@
 **This has to be in place to be able to create the new postgres V11 DB container.**
```

Comment:
Shall we change the page title to "Steps to migrate existing v9.6 PostgreSQL database to v11"

Reply:
updated